packages/number/package.json

```diff
@@ -26,11 +26,11 @@
 		"access": "public"
 	},
 	"scripts": {
-		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && npx rimraf dist",
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && pnpm exec rimraf dist",
```

Comment:
Ah, yes, we should fix it! Updated in 63e02b60f7617a1e0f7045c401f83d5f20932be1. Thanks!

Reply:
> I tried adding root: true to .eslintrc.js and it fixed the error. I don't think this issue is related to your PR, but perhaps it may be beneficial for us to add root: true to our eslintrc to enforce it to use the correct config file?

Sounds good. 👍 Added in f93f254a671064c159035a725def10fe2cea1035.

Reply:
> This isn't related to pnpm either, pretty sure it's related to my mac's configuration somehow. I've tried to increase my mac's opened file limit from 256 to 1028 with ulimit -n, but it didn't help. I'll revisit this soon.

@ilyasfoo I just checked my Mac. My opened file limit setting is also 256. 🤔 

However, I think this might be because we have more files now in `packages/**/node_modules`. AFAIK, babel default only ignore root `node_modules`, not `packages/**/node_modules`. I think maybe we should exclude them from babel transpiling as well. What do you think?

Reply:
Yup we should exclude them from babel transpiling

Reply:
> Yup we should exclude them from babel transpiling

Thanks for confirming! Updated in 294fa4a9ee2072780678d124231bfe1663ec9113.

Reply:
Thanks, @chihsuan! I confirmed `pnpm run lint` and `npm run i18n:js` works for me now 👍